docs/how-to/add-passkey.md

```diff
@@ -167,23 +167,24 @@ The **EMAIL** delivery method sends an email to the user. Clicking the link will
 2. Call the endpoint in your application following the below code example. 
```

Comment:
nit: 2 and 3 don't seem to be properly indented

Reply:
Hm what are you seeing?

Reply:
If you look at 1, you'll notice everything is indented. If you look straight down from 1, you see 2. That is not the case with 2 and 3.

![Screenshot 2023-08-11 at 12 22 48 PM](https://github.com/gobeyondidentity/developer-docs/assets/4763133/9b74d880-adaf-496a-ac96-3fb6dc367d73)
![Screenshot 2023-08-11 at 12 23 01 PM](https://github.com/gobeyondidentity/developer-docs/assets/4763133/e3deacb8-ad16-43d1-be40-c60356ff5eea)

Reply:
Hm so I used a piece of paper to line them up and it looks like the numbers are actually ok. But what you might be seeing it the indentation of some content indented under the number vs some not indented under the number giving a weird effect. I think this was what Patricia was trying to do was to make sure we indented all content nested under numbers.

Reply:
I fixed indented some of the content, but it looks like it doens't work for blocks :::note or :::tip or imported content from includes.